iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -274,6 +288,36 @@ TEST_F(PortManager_test, InterfaceAndApplicationsOverflow)
             iox::capro::Interfaces::INTERNAL, iox::ProcessName_t(iox::cxx::TruncateToCapacity, newItfName));
         EXPECT_THAT(interp, Ne(nullptr));
     }
+
+    // test if overflow errors get hit
+    {
+        auto errorHandlerCalled{false};
+        auto errorHandlerGuard = iox::ErrorHandler::SetTemporaryErrorHandler(
+            [&errorHandlerCalled](const iox::Error, const std::function<void()>, const iox::ErrorLevel) {
+                errorHandlerCalled = true;
+            });
+
+        auto interfacePointer = m_portManager->acquireInterfacePortData(iox::capro::Interfaces::INTERNAL, "itfPenguin");
+        EXPECT_THAT(interfacePointer, Eq(nullptr));
+        EXPECT_TRUE(errorHandlerCalled);
```

Comment:
in the `PublisherOverflow` the correct error code is also tested. Shouldn't this be done here too?

Reply:
Not possible since this returns just a pointer and not an expected. There is a todo in the code to change this.